### PR TITLE
[Rule Tuning] Several rule tunings

### PR DESCRIPTION
**Pull request by Aegrah**
## Summary
Several rule tunings of rules that still generated a bit too much noise after last tuning

### CODE REVIEW

File: rules/linux/impact_potential_linux_ransomware_file_encryption.toml [hunk 1]
```diff
@@ -25,20 +25,18 @@ severity = "medium"
...
-      "php", "pyc", "log", "bak", "bin", "csv", "pdf", "cfg", "*old"
-      ) and not 
-    process.name : (
-      "dpkg", "yum", "dnf", "rpm", "dockerd", "go", "java", "pip*", "python*", "node", "containerd", "php", "p4d", 
-      "conda", "chrome", "imap", "cmake", "firefox", "semanage", "semodule", "ansible-galaxy"
-    ) ] with runs=100 | tail 1
+sequence by host.id, process.entity_id with maxspan=1s
+  [file where host.os.type == "linux" and event.type == "change" and event.action == "rename" and file.extension : "?*" 
+   and ((process.name in ("bash", "dash", "sh", "tcsh", "csh", "zsh", "ksh", "fish", "ash", "openssl")) or
+   (process.executable : ("./*", "/tmp/*", "/var/tmp/*", "/dev/shm/*", "/var/run/*", "/boot/*", "/srv/*","/run/*"))) and
```
Comment (Aegrah) [inline rules/linux/impact_potential_linux_ransomware_file_encryption.toml:31]:
```suggestion
   (process.executable : ("./*", "/tmp/*", "/var/tmp/*", "/dev/shm/*", "/var/run/*", "/boot/*", "/srv/*", "/run/*"))) and
```

File: rules/linux/impact_potential_linux_ransomware_note_detected.toml [hunk 2]
```diff
@@ -27,23 +27,20 @@ tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic:
...
-      "dpkg",  "yum", "dnf", "rpm", "dockerd"
-      ) ] with runs=100
-  [ file where host.os.type == "linux" and event.action == "creation" and file.extension == "txt" and 
-    file.name : (
-      "*crypt*", "*restore*", "*lock*", "*recovery*", "*data*", 
-      "*read*", "*instruction*", "*how_to*", "*ransom*"
-      ) and not process.name : ("go", "java", "pip*", "python*", "node", "containerd") ] | tail 1
+  [file where host.os.type == "linux" and event.type == "change" and event.action == "rename" and file.extension : "?*" 
+   and ((process.name in ("bash", "dash", "sh", "tcsh", "csh", "zsh", "ksh", "fish", "ash", "openssl")) or
+   (process.executable : ("./*", "/tmp/*", "/var/tmp/*", "/dev/shm/*", "/var/run/*", "/boot/*", "/srv/*","/run/*"))) and
```
Comment (Aegrah) [inline rules/linux/impact_potential_linux_ransomware_note_detected.toml:32]:
```suggestion
   (process.executable : ("./*", "/tmp/*", "/var/tmp/*", "/dev/shm/*", "/var/run/*", "/boot/*", "/srv/*", "/run/*"))) and
```

**Comment by Aegrah** [comment]
Moved a few of the auditd_manager rules that I created over to building block rules to make sure they won't end up being too noisy. Can move them back to DR once I'm sure they aren't noisy.

Applied tunings to a large set of Linux rules, tuning results are starting to look very good.

File: rules/linux/impact_potential_linux_ransomware_file_encryption.toml [hunk 3]
```diff
@@ -25,20 +25,18 @@ severity = "medium"
...
+  [file where host.os.type == "linux" and event.type == "change" and event.action == "rename" and file.extension : "?*" 
+   and ((process.name in ("bash", "dash", "sh", "tcsh", "csh", "zsh", "ksh", "fish", "ash", "openssl")) or
+   (process.executable : ("./*", "/tmp/*", "/var/tmp/*", "/dev/shm/*", "/var/run/*", "/boot/*", "/srv/*", "/run/*"))) and
+   file.path : (
+   "/home/*/Downloads/*", "/home/*/Documents/*", "/root/*", "/bin/*", "/usr/bin/*",
+   "/opt/*", "/etc/*", "/var/log/*", "/var/lib/log/*", "/var/backup/*", "/var/www/*") and not ((
+   process.name : (
+   "dpkg", "yum", "dnf", "rpm", "dockerd", "go", "java", "pip*", "python*", "node", "containerd", "php", "p4d",
+   "conda", "chrome", "imap", "cmake", "firefox", "semanage", "semodule", "ansible-galaxy", "fc-cache", "jammy", "git",
+   "systemsettings", "vmis-launcher")) or (file.path : "/etc/selinux/*") or (file.extension in ("qmlc", "txt")
```
Comment (Aegrah) [inline rules/linux/impact_potential_linux_ransomware_file_encryption.toml:38]:
```suggestion
   "systemsettings", "vmis-launcher")) or file.path : "/etc/selinux/*" or (file.extension in ("qmlc", "txt")
```

File: rules/linux/discovery_linux_hping_activity.toml [hunk 4]
```diff
@@ -20,19 +20,20 @@ false_positives = [
...
-severity = "high"
+severity = "medium"
 tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Discovery", "Data Source: Elastic Endgame"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and host.os.type:linux and event.type:(start or process_started) and process.name:(hping or hping2 or hping3)
+process where host.os.type == "linux" and event.type in ("start", "process_started")
```
Comment (w0rk3r) [inline rules/linux/discovery_linux_hping_activity.toml:35]:
```suggestion
process where host.os.type == "linux" and event.type == "start"
```

Comment (w0rk3r) [reply rules/linux/discovery_linux_hping_activity.toml:35]: https://github.com/elastic/detection-rules/pull/2238

File: rules/linux/discovery_linux_nping_activity.toml [hunk 5]
```diff
@@ -29,10 +29,10 @@ rule_id = "0d69150b-96f8-467c-a86d-a67a3378ce77"
 severity = "medium"
 tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Discovery", "Data Source: Elastic Endgame"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and host.os.type:linux and event.type:(start or process_started) and process.name:nping
+process where host.os.type == "linux" and event.type in ("start", "process_started") and process.name == "nping"
```
Comment (w0rk3r) [inline rules/linux/discovery_linux_nping_activity.toml:35]:
```suggestion
process where host.os.type == "linux" and event.type == "start" and process.name == "nping"
```

**Review by w0rk3r — APPROVED** [review]

**Pull request merged**